packages/feedback-service/package.json

```diff
@@ -29,61 +29,65 @@
     "start": "webpack --watch --config webpack.dev.js"
   },
   "author": {
-    "name": "Anjnee K Sharma",
-    "email": "anjsharm@redhat.com"
+    "name": "RIgin Oommen",
+    "email": "riginoommen@gmail.com"
   },
   "contributors": [],
   "license": "MIT",
   "dependencies": {
-    "apollo-log": "0.2.0",
-    "apollo-server-express": "2.19.2",
-    "clean-webpack-plugin": "3.0.0",
-    "debug": "4.3.1",
+    "apollo-log": "^0.2.0",
+    "apollo-server-express": "^2.19.0",
+    "body-parser": "^1.19.0",
+    "clean-webpack-plugin": "^3.0.0",
+    "cookie-parser": "^1.4.4",
+    "cors": "^2.8.5",
+    "dotenv": "^8.2.0",
     "dotenv-safe": "8.2.0",
-    "ejs": "3.1.5",
-    "errorhandler": "1.5.1",
-    "express": "4.17.1",
-    "graphql": "15.4.0",
-    "graphql-iso-date": "3.6.1",
-    "graphql-redis-subscriptions": "2.3.1",
+    "dotenv-webpack": "^1.7.0",
```

Comment:
Remove dotenv-webpack. Use `dotenv-safe` instead. Refer notifications-service for the env setup.